harness/determined/pytorch/_trainer.py

```diff
@@ -289,4 +291,10 @@ def init(
             debug_enabled=debug_enabled,
         )
 
+        # Tensorboard logging is enabled by default
+        # We only need to call set_enable_tensorboard_logging on the trial object of user pass in
+        # enable_tensorboard_logging = False
+        if not enable_tensorboard_logging:
```

Comment:
pass this into the `PyTorchTrialContext` constructor directly? part of the reason to move to `init` is to keep the context initialization in the same place.